src/app/main.cpp

```diff
@@ -124,11 +127,32 @@ int main(int argc, char *argv[])
     setupDpi();
 #endif
 
+    // we need a copy of the arguments to extend them with the params file contents (see right below)
+    std::vector<char*> arguments;
```

Comment:
Shouldn't you free the vector elements when it is about to go out of scope?

Reply:
I don't know how to do that elegantly. There are many exit points and I'd have to add a RAII guard here. But the heap memory will be freed anyway on process exit, so it is OK, IMO, to leave that to the system.

Reply:
Added the shortest solution I could come up with in its own commit.